be/server.go

```diff
@@ -9,9 +9,10 @@ import (
 )
 
 const (
-	EndPointUser   = "/update_or_create_user"
-	EndPointReport = "/report"
-	EndPointGetMap = "/get_map"
+	EndPointUser          = "/update_or_create_user"
+	EndPointPrivacyAndTOC = "/update_privacy_and_toc"
+	EndPointReport        = "/report"
+	EndPointGetMap        = "/get_map"
```

Comment:
If you are trying to make it in alphabetic order, then GetMap goes first and user last.

Reply:
It's in a functionality order. User and Privacy are grouped together.